docker-app/qfieldcloud/core/cron.py

```diff
@@ -52,16 +53,12 @@ class SetTerminatedWorkersToFinalStatusJob(CronJobBase):
     code = "qfieldcloud.set_terminated_workers_to_final_status"
 
     def do(self):
-
-        client = docker.from_env()
-        qgis_containers = client.containers.list(
-            sparse=True, filters={"ancestor": QGIS_CONTAINER_NAME}
-        )
-        qgis_container_ids = [c.id for c in qgis_containers]
-
         jobs = Job.objects.filter(
             status__in=[Job.Status.QUEUED, Job.Status.STARTED],
-        ).exclude(container_id__in=qgis_container_ids)
```

Comment:
We don't use the container_id anywhere else, do we ? if so shouldn't we drop it ?

Also, could it not now happen that a container now runs, exceeds timeout (not sure if this can happen ?), then its job is set to "FAILED" by the cronjob, and then it finishes (maybe even successfully) ? I guess we can't really kill the container, but maybe in the worker, we could do a last check to ensure the status is still "STARTED" before setting it to "SUCCESSFUL" ?

Maybe not too problematic though, and probably will never happen...